src/main/java/uk/gov/hmcts/reform/ucmc/request/RequestData.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.ucmc.request;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Service
```

Comment:
Shouldn't this be simple POJO or data object rather than a bean?

Reply:
we need to inject HttpServletRequest here